public/js/util/getProductionOffice.js

```diff
@@ -0,0 +1,34 @@
+import moment from 'moment';
+
+function isBetween({value, lower, upper}) {
```

Comment:
The destructuring here is a bit redundant, as there's nothing stopping the calls below just sending three arguments

Reply:
I like it because it it's an easy way to make explicit the argument names in the calls to `isBetween` (but as nice as it is, I can't claim credit for it - I copied this from mam).